client/state/current-user/gravatar-status/reducer.js

```diff
@@ -16,11 +16,7 @@ export const isUploading = createReducer( false, {
 } );
 
 export const tempImage = createReducer( {}, {
-	[ GRAVATAR_UPLOAD_RECEIVE ]: ( state, action ) => {
-		return {
-			src: action.src
-		};
-	}
+	[ GRAVATAR_UPLOAD_RECEIVE ]: ( state, action ) => ( { src: action.src } ),
 } );
```

Comment:
do we really need `createReducer()` here? if all we're doing is this simple little thing, how about making it a simple little function?

```js
export const tempImage = ( state = null, { type, src } ) =>
	type === GRAVATAR_UPLOAD_RECEIVE
		? src
		: state;
```

Also, why are we storing this string inside an object instead of just storing the string? Seems like a bit hasty future-planning? If we _do_ predict needing more properties later on, that's what `combineReducers()` is for, so we can write our reducers as simple functions on simple data types and then compose them together.